ansible/roles/debops.postfix/defaults/main.yml

```diff
@@ -900,7 +900,9 @@ postfix__original_mastercf:
     private: False
     chroot: True
     command: 'smtpd'
-    state: 'comment'
+    state: '{{ "present"  
+               if (postfix__pki|bool)
+               else "comment" }}'
```

Comment:
This is a wrong place to make this change. This variable holds the original `/etc/postfix/master.cf` contents, so that they can be changed by other DebOps roles, in particular `debops.postconf`. In there, the `smtps` aka `submissions` service will be enabled automatically when `auth` capability is enabled as well. You should revert this change.